content/concepts/tokens/nftoken-auctions.md

```diff
@@ -0,0 +1,60 @@
+---
+html: nftoken-auctions.html
+parent: non-fungible-tokens.html
+blurb: You can assign another account to mint NFTs in your stead.
+labels:
+ - Non-fungible Tokens, NFTs
+---
+
+# Running an NFT Auction
+
+There are several ways to run an auction, each with advantages and disadvantages.
+
+## Run the Auction Off the XRPL, Complete the Purchase on XRPL
```

Comment:
```suggestion
## Run the Auction and Complete the Purchase on the XRPL

Reply:
I understand your reasoning, but the main point of this option is that the auction is not facilitated by the XRPL.